app/views/admin/users/show.html.erb

```diff
@@ -0,0 +1,138 @@
+<div id="admin_columns">
+
+  <div class="admin_column_left box">
+    <h2>Admin</h2>
+    <%= render partial: 'admin/navigation' %>
+  </div>
+
+  <div class="admin_column_right">
+    <div class="admin_column_right_inner box">
+      <h3 style="padding-left: 1em;">User</h3>
+      <div style="padding: 0 1em;">
+        <header class="admin_assets_header">
+          <div class="date_column">Date</div>
+          <div class="user_column">User</div>
+          <div class="track_column">Track</div>
+        </header>
+        <div class="admin_users_rows">
+          <div data-controller="admin--user" data-admin--user-deleted="<%= @user.soft_deleted? %>" data-admin--user-spam="<%= @user.is_spam? %>">
```

Comment:
Divergence is almost guaranteed on these pages, as admin stuff is changing rapidly — we need to use layouts/partials for this work (vs. copy and paste).

Reply:
Yes. I was thinking about that.

The only reason I left it as is was because I wasn't sure if we wanted some other layout for `admin/user#show` page or if a collection of three tables is acceptable.

When I was cleaning the UI there were elements (like links on user login in comments and assets table) that were useless and also tell us that it should have a different layout than assets#index. 

Thoughts? Maybe we could prioritize new user#show design soon-ish?